crucible-syntax/test-data/simulator-tests/seq-test1.cbl

```diff
@@ -0,0 +1,20 @@
+(defun @main () Unit
+  (start start:
+    (let n (the (Sequence Nat) seq-nil))
+    (assert! (seq-nil? n) "nil test")
+    (assert! (equal? 0 (seq-length n)) "nil length test")
+
+    (let s1 (seq-cons 5 n))
+
+    (assert! (not (seq-nil? s1)) "cons test")
+    (assert! (equal? 1 (seq-length s1)) "cons length test")
+
+    (let v (from-just (seq-head s1) "head s1"))
+    (let t (from-just (seq-tail s1) "tail s1"))
+    (let u (from-just (seq-uncons s1) "uncons s1"))
```

Comment:
I don't think there's any `assert!`/`print` statement involving this `u` value. I think it would be worthwhile to add one for consistency with the other operations featured in this test case.

Reply:
Absolutely.  Turns out, I have to implement syntax for structs first, which is why I stopped here for now. :-)